File: TabBarController_no_StoryBoard_Obj-c/TabBarController_no_StoryBoard_Obj-c/ViewController.m
//
//  ViewController.m
//  TabBarController_no_StoryBoard_Obj-c
//
//  Created by yoon tae soo on 2021/01/17.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
}

- (id) initWithParams:(NSString *)title bgColor:(UIColor *)bgColor {
    
    self.title = title;
    self.view.backgroundColor = bgColor;
    
    return self;
}


@end

File: NavigationController/CoreData/ViewController.swift
//
//  ViewController.swift
//  CoreData
//
//  Created by yoon tae soo on 2020/12/24.
//

import UIKit

class ViewController: UIViewController {

    var getData : String?
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
        
        view.backgroundColor = .yellow
        
        print(getData ?? "없습니다")
        
        
        
    }


}

